lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -71,4 +72,28 @@ def test_masquerade(self):
         self.update_masquerade(username=user.username)
         self.assertEqual(self.client.get(self.url).data['dates_widget']['user_timezone'], 'Asia/Tokyo')
 
+    @ddt.data(
+        (True, True, True, True),  # happy path
```

Comment:
This is an aggressively happy path. An enrolled staff user is a publicly viewable course.